apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,65 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <section class="hero-header">
```

Comment:
You should revise the structure of the html in general. You should have this as basic structure: 
```
<header></header>
<main></main>
<footer></footer>
```
Instead of having multiple ```header```s insides ```section```s that are inside the ```main```
If you have any questions about this please let us know